compiler/rustc_feature/src/accepted.rs

```diff
@@ -169,6 +169,8 @@ declare_features! (
     (accepted, global_allocator, "1.28.0", Some(27389), None),
     // FIXME: explain `globs`.
     (accepted, globs, "1.0.0", None, None),
+    /// Allows using `..X`, `..=X`, `...X`, and `X..` as a pattern.
```

Comment:
The `...X` form was removed in #68120.

```suggestion
    /// Allows using `..X`, `..=X`, and `X..` as a pattern.
```

Might be good to also update the PR description with the same change, to avoid confusion.

Reply:
As another note, `..X` is not stabilized (it also requires exclusive_range_pattern), which this seems to imply.

The only thing changing here is `..=X`. It might be good to change the PR description to make that a little clearer.

Reply:
I shouldn't have copy/paste the previous comment without verifying it. Fixed.